packages/aws-cdk-lib/aws-apigateway/README.md

```diff
@@ -267,6 +267,10 @@ The following example uses sets up two Resources '/pets' and '/books' in separat
 
 [Resources grouped into nested stacks](test/integ.restapi-import.lit.ts)
 
+> **Warning:** In the code above, an API Gateway deployment is created during the initial CDK deployment.
+However, if there are changes to the resources in subsequent CDK deployments, a new API Gateway deployment is not automatically created. As a result, the latest state of the resources is not reflected.
+To ensure the latest state of the resources is reflected, a manual deployment of the API Gateway is required after the CDK deployment.
```

Comment:
It is good to insert line breaks at visually appropriate places in the document. (We can feel relief that these line breaks are not applied when they are actually displayed in the browser.)

```suggestion
> **Warning:** In the code above, an API Gateway deployment is created during the initial CDK deployment.
However, if there are changes to the resources in subsequent CDK deployments, a new API Gateway deployment is not
automatically created. As a result, the latest state of the resources is not reflected. To ensure the latest state
of the resources is reflected, a manual deployment of the API Gateway is required after the CDK deployment.
```

Reply:
@go-to-k 
Thank you! Made corrections!